jupyterlab_automl/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
This can be dropped in favor of the LICENSE file at the top-level.

Reply:
Sure thing.